xdp-loader/README.org

```diff
@@ -77,6 +77,16 @@ file. The default is to use the first program in each file. Only one of
 --section and --prog-name may be specified. If this option is set, it applies to
 all programs being loaded.
 
+** -P, --prio <priority>
+Specify with BPF program with the run priority to load the XDP program(s) from
+in each file. The default is to use the program metadata of ELF section. If this
+option is set, it applies to all programs being loaded.
+
+** -A, --actions <actions>
+Specify with BPF program with the "chain actions" to load the XDP program(s) from
+in each file. The default is to use the program metadata of ELF section. If this
+option is set, it applies to all programs being loaded.
```

Comment:
As above; here we could replace the first two sentences with: "Specify the "chain call actions" of the loaded XDP program(s). These are the XDP actions that will cause the next program loaded on the interface to be called, instead of returning immediately. The default is to use the value set in the metadata in the program elf file, or XDP_PASS if no such metadata is set."